packages/ckeditor5-ui/src/formheader/formheaderview.ts

```diff
@@ -61,41 +66,69 @@ export default class FormHeaderView extends View {
 	 */
 	constructor(
 		locale: Locale | undefined,
-		options: {
-			label?: string | null;
-			class?: string | null;
-			icon?: string | null;
-		} = {}
+		options: FormHeaderViewOptions = {}
 	) {
 		super( locale );
 
 		const bind = this.bindTemplate;
 
+		this.options = options;
+		this.iconView = this._createIcon();
+
 		this.set( 'label', options.label || '' );
 		this.set( 'class', options.class || null );
 
 		this.children = this.createCollection();
 
+		if ( this.options.left ) {
+			this.children.add( this.options.left );
+		}
+
+		if ( this.iconView ) {
+			this.children.add( this.iconView );
+		}
+
+		this.children.add( this._createLabel() );
+
+		if ( this.options.right ) {
+			this.children.add( this.options.right );
+		}
+
 		this.setTemplate( {
 			tag: 'div',
 			attributes: {
 				class: [
 					'ck',
 					'ck-form__header',
 					bind.to( 'class' )
-				]
+				],
+
+				// https://github.com/ckeditor/ckeditor5-link/issues/90
+				tabindex: '-1'
```

Comment:
Why is this needed? Does the form header should be focusable on its own?

Reply:
Here and in a few other places we use this workaround for an issue where `Escape` stops working, when a given element (header in this case) in clicked on.